composer.json

```diff
@@ -66,9 +66,6 @@
             "Tests/"
```

Comment:
You can try and if you do, please measure the difference, but I feel it won't be that much.